pkg/cableengine/ipsec/driver.go

```diff
@@ -0,0 +1,22 @@
+package ipsec
+
+import (
+	"github.com/submariner-io/submariner/pkg/types"
+)
+
+// Driver is used by the ipsec engine to actually connect the tunnels
+type Driver interface {
+
+	// Initialize the driver with any state it needs
+	Init() error
```

Comment:
As per https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences, the comment  should begin with the name of the function, ie

    // Init performs any initialization needed by the driver.

Reply:
Oh, that's for IDEs to catch which function are you talking about?